### PR TITLE
Add kubernetes partners page

### LINKED ISSUE #3359: Beta: Build Kubernetes partners page
Copy doc: https://docs.google.com/document/d/1vRlHxWr7pxstx5pOtqXWrIviXtdPvW2gd3-mNeYQKp0/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add kubernetes partners page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes/partners>
- See that the page matches the [copy doc](https://docs.google.com/document/d/1vRlHxWr7pxstx5pOtqXWrIviXtdPvW2gd3-mNeYQKp0/edit#heading=h.pxakvoload1i)

## Issue / Card

Fixes #3359

### CODE REVIEW

**Comment by anasereijo** [comment]
@frankiegranato can you please review the ctas/links? not sure if we should use green ctas or links. At the moment we have white.

**Comment by anasereijo** [comment]
there's some new images on the copy doc
screen shot 2018-06-22 at 17 00 16
screen shot 2018-06-22 at 16 53 11

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by anasereijo** [comment]
@steverydz can you please align horizontally the logos with the text? see below 

screen shot 2018-06-22 at 17 20 11

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by anasereijo** [comment]
@steverydz can you please make the CTA's green? (keep as a link on the hero section)

**Comment by steverydz** [comment]
@anasereijo Done

File: templates/kubernetes/partners.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      <img style="max-width: 250px;" src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png" alt="">
+    </div>
+  </div>
+  <div class="row u-vertically-spaced u-equal-height">
+    <div class="col-8">
+      <h3>JFrog Artifactory</h3>
+      <p>JFrog Artifactory is the enterprise-ready Kubernetes registry fully integrated with both Rancher and the Canonical Distribution of Kubernetes (CDK). Long known to developers, Artifactory provides the governance, audit capability and artifact management. Canonical partners with JFrog to integrate CDK with a single point and click installation using conjure-up.</p>
+      <p><a class="p-button--positive" href="https://blog.ubuntu.com/2018/04/12/jfrog-artifactory-and-canonicals-distribution-of-kubernetes">Watch the webinar</a></p>
```
Comment (anthonydillon) [inline templates/kubernetes/partners.html]: Can you please update this link to: https://www.brighttalk.com/webcast/6793/308887

File: templates/kubernetes/partners.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Become a cloud partner</h2>
+      <p>To learn more about becoming a Canonical Kubernetes partner, please contact us today.</p>
+      <p><a class="p-button--positive" href="/partners/contact-us">Get in touch</a> or <a href="https://partners.ubuntu.com/partnering-with-us">learn more about partnering with us&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/kubernetes/partners.html]: Can you update this link to: https://partners.ubuntu.com/contact-us

**Comment by steverydz** [comment]
@anthonydillon Link changes made

**Comment by steverydz** [comment]
@frankiegranato CTAs updated

**Pull request merged**